script/1_0_2current_en_script/Ciel/Day 3/03_01_CIEL03_2_1.txt

```diff
@@ -71,25 +71,25 @@ You seem like a very capable person."
 <Page20>
 "Yes, I try my best not to screw up in front of you. To be honest, I'm actually rather clumsy if anything."
 <Page21>
-...Certainly, it's possible that she may actually be a bit of a klutz?
+...It's certainly possible that she may actually be a bit of a klutz.
 It's the same with her volunteer work.
-People that are good at dealing with things wouldn't do something that doesn't benefit them in the slightest like that.
+Capable individuals wouldn't do something like that which doesn't benefit them in the slightest. // 要領がいい人間 capable might be a bit of a stretch but also connects to line 70
 <Page22>
-"I see. ―――But I like clumsy people.
+"I see. ―――But I actually like clumsy people.
 I feel at ease when I'm with them."
-Arihiko agrees with me.
+Arihiko nods in agreement.
 "So, does a late riser like you normally go to the cafeteria?"
 <Page23>
-"Yes, that's why I eat in the cafeteria most of the time, but sometimes I make my own lunch like this. Usually, it's just a bunch of pre-prepared dishes.
+"Yes, that's why I eat in the cafeteria most of the time, only making my own lunch like this every so often. Usually, it's just a bunch of pre-prepared dishes.
```

Comment:
first change here made because specifying "that's why" after shiki's question is kinda weird. Other change just flow
> Yes, I'm in the cafeteria most of the time. I only make my own lunch like this